op-verifier/deploy_l1/00_op_verifier.ts

```diff
@@ -5,10 +5,14 @@ import fs from 'fs';
 const GATEWAY_URLS = {
   'opDevnetL1':'http://localhost:8080/{sender}/{data}.json',
   'goerli':'https://op-gateway-worker.ens-cf.workers.dev/{sender}/{data}.json',
+  'sepolia':'https://op-sepolia-gateway-worker.ens-cf.workers.dev/{sender}/{data}.json',
+  'sepoliaforbase':'https://base-sepolia-gateway-worker.ens-cf.workers.dev/{sender}/{data}.json',
```

Comment:
we can use `baseSepolia` for convention with the rest of the code.

Reply:
`baseSepolia` means it's a base l2 testnet. `sepoliaforbase` is a sepolia l1 chain but for base contracts

Reply:
ah, I see. then maybe at least have it camelCase?